File: Python/M01_ProgrammingBasics/L06_NestedLoops/Lab/Solutions/P05_Travelling.py
destination = 0
new_savings = 0
savings = 0

while destination != "End":
    destination = input()
    if destination == "End":
        break
    min_budget = float(input())
    while min_budget >= new_savings:
        savings = float(input())
        new_savings += savings
        if new_savings >= min_budget:
            print(f"Going to {destination}!")
            new_savings = 0
            break

File: C++/M01_ProgrammingBasics/L06_NestedLoops/Lab/Solutions/P05_Travelling.cpp
#include <iostream>
using namespace std;

int main() {
    string destination;
    cin >> destination;

    while (destination != "End") {
        double needBudget;
        cin >> needBudget;

        double savedMoney = 0;
        while (savedMoney < needBudget) {
            double sum;
            cin >> sum;
            savedMoney += sum;
        }
        cout << "Going to " << destination << "!" << endl;

        cin >> destination;
    }
    
    return 0;
}
